spec/services/activities/import_from_csv_spec.rb

```diff
@@ -712,6 +712,23 @@
       end
     end
 
+    it "has an error if both planned_start_date and actual_start_date are missing" do
```

Comment:
Do we want to test fror end_dates?

Reply:
Hm, I wondered that, but decided against it because what I'd really want to test is that the validation includes the `dates_step`, the meaning of which is tested elsewhere (in `spec/models/activity_spec.rb`, contexts `"when planned start and actual start dates are blank"`, `"when planned_start_date is blank but actual_start_date is not nil"`, etc.)

Reply:
Cool! I feel like the name reflecting that decision would make a lot of sense.